typescript/packages/orb/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@subconsciousnetwork/orb",
+  "author": "Subconscious Inc.",
+  "license": "Apache-2.0 OR MIT",
+  "version": "0.1.0",
+  "description": "A batteries-included JavaScript package for accessing the Noosphere",
+  "type": "module",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "wireit",
+    "generate_wasm": "wireit",
+    "clean": "wireit",
+    "test": "wireit"
+  },
+  "dependencies": {
+    "@web/dev-server": "^0.1.34",
+    "esbuild": "^0.15.9",
+    "typescript": "~4.8.2",
+    "wireit": "^0.8.0"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "generate_wasm"
+      ],
+      "command": "tsc",
+      "files": [
+        "./src/**/*.ts"
+      ]
```

Comment:
Missing `output` here? If you add it, you'll get incremental build and caching (e.g. if none of the input files changed since the last run, it won't re-run).

Reply:
It's a bit tedious to anticipate all of the outputs in cases like this. Are there any shortcuts to, say, derive outputs automatically from the list of inputs? Like: every input .ts file produces a corresponding output .d.ts and .js file.

Reply:
Having a dedicated output folder could be a simple solution, since then you can just put e.g. `lib`

I have been considering a special mode where it runs each script in isolation, looks for new files, and then suggests updates to your config (and/or warns you if there are undocumented outputs).

Reply:
The challenge in this particular case is that some of what ends up in `lib` is produced by `wasm-bindgen`. For now, I'm manually listing all the outputs.